Source/Core/arrayFill.js

```diff
@@ -0,0 +1,64 @@
+/*global define*/
+define([
+        './DeveloperError',
+        './defaultValue',
+        './defined'
+    ], function(
+        DeveloperError,
+        defaultValue,
+        defined) {
+    'use strict';
+
+    /**
+     * Fill an array or a portion of an array with a given value.
+     * @private
```

Comment:
I could be wrong, but if the doc build breaks it's because @private has to be the last thing in the doc (If I recall correctly).